apps/web/src/pages/templates/workflow/digest/WillBeSentHeader.tsx

```diff
@@ -0,0 +1,175 @@
+import { useCallback, useEffect } from 'react';
+import { useFormContext } from 'react-hook-form';
+import * as capitalize from 'lodash.capitalize';
+import { useMantineColorScheme } from '@mantine/core';
+import { DigestTypeEnum, DigestUnitEnum, MonthlyTypeEnum } from '@novu/shared';
+
+import { colors } from '../../../../design-system';
+import { pluralizeTime } from '../../../../utils';
+
+const getOrdinalValueLabel = (value: string) => {
+  if (value === 'day' || value === 'weekday') {
+    return value;
+  }
+  if (value === 'weekend') {
+    return 'weekend day';
+  }
+
+  return capitalize(value);
+};
+
+const getOrdinal = (num: string | number) => {
+  if (typeof num === 'string') {
+    const res = parseInt(num, 10);
+
+    if (isNaN(res)) {
+      return num;
+    }
+    num = res;
+  }
+  const ord = ['st', 'nd', 'rd'];
+  const exceptions = [11, 12, 13];
+  let nth = ord[(num % 10) - 1];
+
+  if (ord[(num % 10) - 1] == undefined || exceptions.includes(num % 100)) {
```

Comment:
```suggestion
  if (nth == undefined || exceptions.includes(num % 100)) {
```
You have it already calculated.